### ISSUE TITLE
Question in calendar  >> when I set up a reminder in a todo, I dont receive any notification

### ISSUE THREAD

**Issue by charles-rollet** [hcengineering/platform#6668]
Question in calendar  >> when I set up a reminder in a todo, I dont receive any notification

<https://front.hc.engineering/workbench/hulydemo/team>

I) first I set up the calendar notification by enabling all the reminder notifications as below: 

![Capture d’écran 2024-09-22 à 16.28.32.png](https://front.hc.engineering/files/bb793ec1-252e-4419-8998-7f220ababdfa/b743e36a-c799-467d-84f8-62c271f225e2?file=b743e36a-c799-467d-84f8-62c271f225e2&width=977&token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHRyYSI6eyJzZXJ2aWNlIjoiZ2l0aHViIiwibGlua0lkIjoiYjA5YjI4NmItNWExZi00ZWQwLTlhZDItNGZhM2M0YzFhZmQwIiwiZ3Vlc3QiOiJ0cnVlIn0sImFjY291bnQiOiJiMDliMjg2Yi01YTFmLTRlZDAtOWFkMi00ZmEzYzRjMWFmZDAiLCJ3b3Jrc3BhY2UiOiJiYjc5M2VjMS0yNTJlLTQ0MTktODk5OC03ZjIyMGFiYWJkZmEifQ.mP_hzS4SSW-9AG4BlxQfcifzkMRiL_uFueR6eWTt5v8 "Capture d’écran 2024-09-22 à 16.28.32.png")

II) then set a reminder to a todo (to remind me 10 min before)

![Capture d’écran 2024-09-22 à 16.29.26.png](https://front.hc.engineering/files/bb793ec1-252e-4419-8998-7f220ababdfa/326a9411-f7c5-4299-9fc6-0e1d3d53f3d4?file=326a9411-f7c5-4299-9fc6-0e1d3d53f3d4&width=571&token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHRyYSI6eyJzZXJ2aWNlIjoiZ2l0aHViIiwibGlua0lkIjoiYjA5YjI4NmItNWExZi00ZWQwLTlhZDItNGZhM2M0YzFhZmQwIiwiZ3Vlc3QiOiJ0cnVlIn0sImFjY291bnQiOiJiMDliMjg2Yi01YTFmLTRlZDAtOWFkMi00ZmEzYzRjMWFmZDAiLCJ3b3Jrc3BhY2UiOiJiYjc5M2VjMS0yNTJlLTQ0MTktODk5OC03ZjIyMGFiYWJkZmEifQ.mP_hzS4SSW-9AG4BlxQfcifzkMRiL_uFueR6eWTt5v8 "Capture d’écran 2024-09-22 à 16.29.26.png")

Expected: to receive 3 notifications 10 min before the todo start time

Actual: nothing happened 

Maybe I did something wrong - would you be able to help or I loop in the devs? 

looping in [Andrey Platov](https://front.hc.engineering/browse/?workspace=bb793ec1-252e-4419-8998-7f220ababdfa&_class=contact%3Aclass%3APerson&_id=62c40aa9e5a6ae23d8307df0&label=Andrey%20Platov) too 😉

**Comment by AbigailDawson** [comment]
Yes this does appear to be a bug, reminder notifications are not working.

**Comment by haiodo** [comment]
Reminders are not working yet ,yes.

**Comment by wafelmolt** [comment]
Hello. Could this be reprioritized. This seams to be an MVP feature for teams organizations in a virtual office : Not having reminders, especially on calendar items, is congesting because, as users, we need to make "manual reminders" per se, sending message to message groups.

**Comment by wafelmolt** [comment]
Hello. Any news on this subject ? Not having notification on meeting's reminders completely defeats the purpose of managing meetings in Huly.
At least, remove the feature in the UI for now so we are not subject to think it's a thing. Thx 

PS: I don't see the HC Engineering Huly link in this thread ? Is the subject too old ? Should we recreate or reference this in a new issue ?

**Comment by wafelmolt** [comment]
Hello. Can we have an ETA for this feature ?
